docs/_articles/en/user-guide/remote-development.md

```diff
@@ -30,35 +35,90 @@ To start remote development in dev container, install:
+
    VS Code detects the dev container configuration file and prompts you to reopen the project folder in a container. If the pop-up window disappears, from the Command Palette, run **Remote-Containers: Reopen in Container**.
-   
+
    VS Code creates a dev container the first time you open the project. After the dev container is built, the project folder in your local system automatically connects and maps to the container, and the side bar shows `Dev Container: Salesforce Project`. The container pre-installs and configures Java, Git, Salesforce CLI, and all other extensions defined in the `devcontainer.json` file.
-   
+
    ![Dev Container](./images/devcontainer.png)
-   
+
 1. From the Command Palette, run the **SFDX: Authorize a Dev Hub** in the container. From the Output panel (below the editor region), you can get the user code and the verification URL required to complete the authorization.
 
    If you are logged in to an org that you don’t want to authorize as a Dev Hub, make sure to log out. Otherwise, you're not prompted to enter the credentials for the org that you want to authorize. After the authorization is complete, the message `SFDX: Authorize a Dev Hub successfully ran` is displayed. In case the success message is not displayed, check if you have the correct user code and try again. If the login page doesn’t prompt you to re-enter the code, terminate the command and run it again.
 
 If you are interested, read more about [Device Authentication Flow](https://help.salesforce.com/articleView?id=remoteaccess_oauth_device_flow.htm&type=5).
-   
-   ![Authorize Success Message](./images/authorize_message.png)
-   
-1. Run **SFDX: Create a Default Scratch Org**.   
+
+![Authorize Success Message](./images/authorize_message.png)
+
+1. Run **SFDX: Create a Default Scratch Org**.
 1. Click **Reopen in Container** to build a dev container. Now the side bar shows `Dev Container: Salesforce Project`.
 
    You’ve set up a dev container for an existing project to use as your full-time development environment.
 
+## Using Windows Subsystem for Linux
+
+For remote development using the Windows Subsytem for Linux follow the instructions below.
+
+## Install
+
+To start developing in a container with the Windows Subsystem for Linux (WSL), first you must set up WSL on your machine. To do so, following the [instructions provided by Microsoft](https://docs.microsoft.com/en-us/windows/wsl/install-win10). **Note, you should only use WSL2. There are known issues with WSL1. The instructions below assume Ubuntu. Other Linux distros should work, but the steps may be slightly different.**
+
+After you have installed WSL2, you will need to setup your environment for Salesforce Development. The two main things you will need are Node.js, the Salesforce CLI, and the JDK.
```

Comment:
The problem I see here is that the Salesforce CLI is not officially supported for running in WSL2. Has that changed ?

Reply:
WSL2 is - it is actually just Linux, so I really don't think there should be an issue. WSL1 was more problematic. In fact, we should probably change what we say about WSL not being supported and say that WSL1 isn't supported, but WSL2 is. //cc @clairebianchi